app/AndroidManifest.xml

```diff
@@ -110,26 +110,26 @@
 
         <provider
             android:name=".odk.provider.FormsProvider"
-            android:authorities="org.commcare.android.provider.odk.forms" />
+            android:authorities="${applicationId}.provider.odk.forms" />
         <provider
             android:name=".odk.provider.InstanceProvider"
-            android:authorities="org.commcare.android.provider.odk.instances" />
+            android:authorities="${applicationId}.provider.odk.instances" />
```

Comment:
Default `applicationId` is `org.commcare.dalvik`, so this will change the overall  `android:authorities="org.commcare.android.provider.odk.instances"` to
`android:authorities="org.commcare.dalvik.provider.odk.instances"`

Not really sure the effects of this change.